<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Drone Updater</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>671</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="serialPortList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ConnectBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QComboBox" name="baudRateBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>0</y>
       <width>104</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="dataBitsBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>104</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="parityBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>104</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="stopBitsBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>104</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>301</width>
       <height>131</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="description">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="manufacturer">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="serial_number">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="location">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="Update_list">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Update list</string>
     </property>
    </widget>
    <widget class="QComboBox" name="flowControlBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>104</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="Console" name="uiConsole">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>321</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Console</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">console.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
